nixos/modules/services/networking/gobgpd.nix

```diff
@@ -0,0 +1,73 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.gobgpd;
+  format = pkgs.formats.toml { };
+in {
+  options.services.gobgpd = {
+    enable = mkEnableOption "Whether to enable GoBGP Routing Daemon.";
```

Comment:
You'll find that some of the text here is redundant. Please take a look at the [definition](https://github.com/NixOS/nixpkgs/blob/e2080b39e60b5d5113d6c226e49d946ce32b1a25/lib/options.nix#L97) of `mkEnableOption`.

Reply:
I understand. Thank you!